src/SprykerEco/Yves/Payone/Dependency/Client/PayoneToCalculationBridge.php

```diff
@@ -29,7 +29,7 @@ public function __construct($calculationClient)
      *
      * @return \Generated\Shared\Transfer\QuoteTransfer
      */
-    public function recalculate(QuoteTransfer $quoteTransfer)
+    public function recalculate(QuoteTransfer $quoteTransfer): QuoteTransfer
```

Comment:
Please don't add return types to Bridges and their interfaces

Reply:
Fixed.